bankaccount.h

```diff
@@ -0,0 +1,36 @@
+#ifndef BANKACCOUNT_H
+#define BANKACCOUNT_H
+
+#include <QObject>
+#include <QString>
+
+class BankAccount : public QObject
+{
+    Q_OBJECT
+    // TODO: declare QString holderName property that's represents account holder name
+    //       this property must have getter and setter
+    Q_PROPERTY(QString m_holderName READ holderName WRITE setHolderName)
+
+
+    // TODO: declare long balance property
+    Q_PROPERTY(qint64 m_balance READ balance)
```

Comment:
1. Not binded to member, this name breaks coding conventions (please check documentation)
2. Not constant